src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin watch-size($size) {
+  width: #{$watch-size};
+  height: #{$watch-size};
+}
+
+@mixin center-size($size) {
+  width: #{$center-size};
+  height: #{$center-size};
+}
```

Comment:
Both of these mixins do the same logic
If you need to create square, name it square, if you need circle, name it circle and add border-radius inside...

Reply:
done